src/rules/order-properties.ts

```diff
@@ -96,11 +104,14 @@ export const rule = createRule<Options>({
 									result,
 								);
 							},
-							message:
-								"Package top-level properties are not ordered in the npm standard way. Run the ESLint auto-fixer to correct.",
-							node: context.sourceCode.ast,
+							loc: properties[i].loc,
+							message: `Package top-level property ${value} is not ordered in the npm standard way. Run the ESLint auto-fixer to correct.`,
```

Comment:
Nit: especially if there are keys with spaces in them, this makes for an easier report.

```suggestion
							message: `Package top-level property "${value}" is not ordered in the npm standard way. Run the ESLint auto-fixer to correct.`,
```